plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/NewNativeAppEngineStandardProjectTest.java

```diff
@@ -84,7 +86,7 @@ private void createAndCheck(String projectName, String packageName, AppEngineRun
     IFacetedProject facetedProject = ProjectFacetsManager.create(project);
     assertNotNull("Native App Engine projects should be faceted", facetedProject);
 
-    if (runtime == null || runtime == AppEngineRuntime.STANDARD_JAVA_7) {
+    if (runtime == AppEngineRuntime.STANDARD_JAVA_7) {
```

Comment:
The intention was for `runtime == null` to mean the _don't change the default option in the wizard_.  Since you're changing the wizard default, this line should be:
```
if(runtime == null || runtime == AppEngineRuntime.STANDARD_JAVA_8) {
```